themes/default/content/docs/intro/concepts/config.md

```diff
@@ -284,7 +284,20 @@ public static void stack(Context ctx) {
 
 {{< /chooser >}}
 
-## Special Configuration Options
+## Provider Configuration Options
+
+There are three ways to configure providers:
+
+1. Set configuration keys in the stack configuration file: `pulumi config set [PROVIDER]:[KEY] [VALUE]`
+2. Set a provider-specific environment variable
+3. Pass arguments to the provider's SDK constructor, in your program
+
+Please note:
+
+* Configuration file settings are only used by the default provider. If you instantiate a provider object, it will not read values from the stack configuration.
```

Comment:
Cross link "provider" to the documentation on Provider's (and possibly capitalize and format as code `Provider`?)

Cross link default provider to documentation on default providers?